### PR TITLE
Rotate orientation of axes 

### PR DESCRIPTION
**Pull request by lauramurgatroyd**
Rotates axes as follows:
image

### CODE REVIEW

File: Wrappers/Python/ccpi/viewer/CILViewer2D.py [hunk 1]
```diff
@@ -1555,6 +1553,9 @@ def installPipeline(self):
         self.ren.ResetCamera()
         self.ren.Render()
 
+        self.camera.SetViewUp(0,-1,0)
+        self.camera.Azimuth(180)
```
Comment (paskino) [inline Wrappers/Python/ccpi/viewer/CILViewer2D.py:1557]: I suppose this should go in `AdjustCamera`?

**Review by paskino — APPROVED** [review]
I think this is fine, however in the future I'd like to be able to implement a more generic way to change the camera position and `viewUp` so that user can change it.

**Comment by lauramurgatroyd** [comment]
I agree that we need to allow the user to set the orientation on each axis.
I started attempting it but managed to break a lot of things so started again and made this PR instead.
I added details of what I would like to do here: https://github.com/vais-ral/CILViewer/issues/104

File: Wrappers/Python/ccpi/viewer/CILViewer2D.py (hunk 1 above)
Comment (lauramurgatroyd) [reply Wrappers/Python/ccpi/viewer/CILViewer2D.py:1557]: This only happens in `installPipeline`, to make sure the initial camera direction is correct.
Whereas `AdjustCamera` is called in `updatePipeline` as well, where we don't want to move the camera to the xy plane

**Pull request merged**